ovc5/software/camera_device_driver/src/sensor_manager.cpp

```diff
@@ -143,30 +143,64 @@ void SensorManager::sendFrames()
   }
 }
 
+/* JSON Format
+ *
+ * {
+ *  "fps": float   Global capture trigger.
```

Comment:
[Nit] It's called `fps` here but `frame_rate` below.
How do you envision having sensors with different frame rate with this setting? Something we did before (and might work) is to have an integer downsampling factor to apply to the frame rate of slower sensors, for example if you had an imager that runs at 120FPS and you want it to work together with a 30FPS imager you would set the main frame rate to the highest and downsample the second imager by a factor of 4, so they are still synchronized.
What do you think?

Reply:
We were discussing setting up a timer for each camera individually. This sounds like the best solution as it will allow each camera to function independently or synchronously.

Reply:
It's a tradeoff between flexibility and making common use cases easy to implement out of the box. For most cases of sensor fusion people would want sensors to be synchronized, I guess that, for example in your case, a typical user would be more concerned about having the synchronization rather than the exact frame rate, for example if they had a 100FPS camera they would probably prefer a 33.33 FPS secondary sensor that is perfectly synchronized, rather than a 30FPS that is not. After all one of the main benefits of OVC is the accurate timing and imager synchronization, otherwise a solution where you buy a bunch of USB / MIPI cameras and connect them to your machine is a lot simpler to implement.
A possibility could also be it being a parameter if you want maximum flexibility, like a boolean to mark a sensor as free running (i.e. arbitrary FPS) or as synchronized (integer multiple / divider).

Reply:
Hmmmm... I think the concern here is that if you have 2 cameras (a high speed and low speed), unless you reduce the low speed to the same exposure time as the high speed, the frames will technically not be aligned (as one will have information past the other one). This will be very obvious in the case of motion or blinking lights/other periodic things. I mostly imagined synchronizing everything at the same frame-rate to avoid the complication but I can see the potential for more information being valuable.